ruby-ubuntu16/Dockerfile

```diff
@@ -77,6 +77,7 @@ RUN apt-get update -y \
         tzdata \
     && apt-get install -y -q --no-install-recommends libgit2-dev \
     && apt-get clean \
+    && apt-get upgrade -yq \
```

Comment:
Shouldn't this happen before `apt-get clean`?

Reply:
Yep. Fixed.